README.md

```diff
@@ -171,3 +171,26 @@ egg.
 You can use the `GITHUB_TOKEN` environment variable to specify the token
 
     $ GITHUB_TOKEN=xxx python -m tools.copy_maintainers
+
+## Integration with `pre-commit`
+
+In any addons repo, you can use these pre-commit hooks:
+
+```yaml
+# .pre-commit-config.yaml file
+repos:
+  - repo: https://github.com/OCA/maintainer-tools
+    rev: master # Better use a tag/commit!
```

Comment:
With pre-commit using tags (or can we use commit hashes?) is a must.

Reply:
OK, I changed it.